superset/views/core.py

```diff
@@ -2161,6 +2161,8 @@ def sql_json(self):
         sql = request.form.get('sql')
         database_id = request.form.get('database_id')
         schema = request.form.get('schema') or None
+        template_params = json.loads(
+            request.form.get('templateParams') or '{}')
```

Comment:
`get` takes a default parameter:

```python
template_params = json.loads(request.form.get('templateParams', '{}'))
```

Reply:
The intent of the `or` is to handle an empty string, though I could probably trim